docs/framework/database/schema-design.md

```diff
@@ -0,0 +1,30 @@
+
+## Aleternate Methods to ERDs to explore CiviCRM's Database Design
+
+Another method ofr viewing CiviCRM's Database model is to use one of various MySQL coding tools e.g. PHPMyAdmin or [MySql WOrkbench](https://www.mysql.com/products/workbench/). These tools allow coders to inspect the Table Structure more easily and look at all the constraints attached to tables very easily. 
+
+## MySQL Workbench Workflow
+
+1. Look at the list of tables in the database. The table name generally gives you a good idea of the type of data it stores.
+2. Pick one table and look closer. Let's look at `civicrm_country` (a relatively simple example) by opening the "Table inspector" for that table.
+  - We can look at the list of columns in the table.
+  ![Mysql Workbench CiviCRM Country Columns](/img/mysql_workbench_civicrm_country_tables.png)
+  We can see that columns have comments which explain (to some extent) the meaning of the data stored in the columns. From the Column Names and the comments we can start to get some idea of the relationship between tables. For example we can see that there is a column `address_format_id` which indicates to us that the table `civicrm_country` has a relationship to the table `civicrm_address_format`
+  - We can also Look at the Foriegn Keys that are relevant to this table
+  ![Mysql Workbench CiviCRM Country Foreign Keys](/img/mysql_workbench_civicrm_country_foreign_keys.png)
+  Here we can get more of a comprenshive picture of what tables `civicrm_country` relates to. We can tell that because we see not only what tables are referenced by `civicrm_country` but also the tables that reference `civicrm_country`. 
+
+## General Characteristics of CiviCRM Tables
+
+When we look at CiviCRM Tables there are a few patterns that generally hold true:
+- Every table has a id column which is Auto Increment and therefore unique key for that table
+- Columns which reference other tables generally speaking will be named in the format of `other table name` + `_id`. For example in `civicrm_country` there is `address_format_id` which indicates that is references `civicrm_address_format.id`
+- Many-to-many relationships use "join tables" as intermediary tables. For example, a contact can have many activity records, and an activity can have many contact records. So the table `civicrm_activity_contact` is used as the glue because it has foreign keys to both.
+- In some places CiviCRM defines schema using a construct called [pseudoconstants](framework/database/schema-definition/#table-field-pseudoconstant) which produces some slightly more complex logic
```

Comment:
Need to change this URL to follow the [standards](https://docs.civicrm.org/dev/en/latest/documentation/markdown/#internal-url-standards)